2fa/README.md

```diff
@@ -2,8 +2,46 @@
 
 ## What is it?
 
+Two factor authentication (2fa from now on) is a method to double check that you're really you by requiring one more check with something you have. This is usually a code generated by an authenticator application or a physical device, or a text message that's sent to your phone.
```

Comment:
Might be worth starting with something well-known. E.g.:

Most people use passwords to access services and information that needs to be kept secure. A password is sometimes called a piece of "something you know" security. However this password could be stolen or guessed by a malicious third party.

Two factor authentication (2FA from now on) is a method to double check that you're really you by requiring additional security information using "something you have". This is usually a code generated by an authenticator application, or a physical device, or a message that's sent to your phone.

For an example imagine you wanted to unlock your car, and it required 2FA. You'd need a password ("something you  know") AND a key ("something you have"). This greatly increases the security of your car.